hosts/ponkila-ephemeral-gamma/default.nix

```diff
@@ -0,0 +1,26 @@
+{ pkgs, config, inputs, lib, ... }:
+{
+  # User options
+  user = {
+    authorizedKeys = [
+      "ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBNMKgTTpGSvPG4p8pRUWg1kqnP9zPKybTHQ0+Q/noY5+M6uOxkLy7FqUIEFUT9ZS/fflLlC/AlJsFBU212UzobA= ssh@secretive.sandbox.local"
+      "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIKEdpdbTOz0h9tVvkn13k1e8X7MnctH3zHRFmYWTbz9T kari@torque"
+      "sk-ssh-ed25519@openssh.com AAAAGnNrLXNzaC1lZDI1NTE5QG9wZW5zc2guY29tAAAAID5aw7sqJrXdKdNVu9IAyCCw1OYHXFQmFu/s/K+GAmGfAAAABHNzaDo= da@pusu"
+      "sk-ssh-ed25519@openssh.com AAAAGnNrLXNzaC1lZDI1NTE5QG9wZW5zc2guY29tAAAAINwWpZR5WuzyJlr7jYoe0mAYp+MJ12doozfqGz9/8NP/AAAABHNzaDo= da@pusu"
+    ];
+  };
+
+  # Localization
+  networking.hostName = "ponkila-ephemeral-beta";
+  time.timeZone = "Europe/Helsinki";
+
+  # SSH
+  services.openssh = {
+    enable = true;
+    settings = {
+      permitRootLogin = true;
+      PasswordAuthentication = false;
+    };
+  };
+  system.stateVersion = "23.05";
```

Comment:
Looking at this stateVersion, one thing that I think we could try (but I don't know how to integrate) is to run stable version of the nixpkgs on platforms on which the support is worse. I think just changing the system state version won't suffice, it's probably something to do with flakes input. But, knowing how to change the flakes nixpkgs source per host basis would probably be useful later on.

Reply:
[`mkIf`](https://nixos.wiki/wiki/NixOS:Properties) could potentially be used to check the system architecture and set the appropriate stateVersion.

Reply:
System architecture is not precise enough, ARM Linux covers [_a lot_ of machines](https://nixos.wiki/wiki/NixOS_on_ARM), and most of these are not UEFI compatible, meaning each requires custom configuration. We won't cover all of them, but making the distinction on architecture level is not a good assumption long-term.